packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC20.tsx

```diff
@@ -40,7 +40,7 @@ export const TransferERC20 = memo<TransferERC20Props>(({ token }) => {
     )
 
     // transfer amount
-    const transferAmount = new BigNumber(amount || '0').multipliedBy(pow10(token.decimals)).toFixed()
+    const transferAmount = formatAmount(amount, token.decimals)
```

Comment:
The format amount is an approximation amount. We need it to be a big int in WEI.